SERVERPLUGINS.md

```diff
@@ -574,7 +574,7 @@ var myFile = require('path').join(app.getDataDirPath(), 'somefile.ext')
 
 ### `app.registerPutHandler(context, path, source, callback)`
 
-If a plugin wants to respond to `PUT` requests for a specific path, it can register an action handler.
+If a plugin wants to respond to HTTP `PUT` requests for a specific path, it can register an action handler.
```

Comment:
We should not say HTTP here, because put requests can come over other protocols.

Reply:
Thanks! Fixed!